slices/libffi7.yaml

```diff
@@ -0,0 +1,8 @@
+package: libffi7
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/lib/*-linux-*/libffi.so.7*:
```

Comment:
nitpicking: no end-of-line after the last line.

Reply:
Thanks ;) done